libs/cardano-ledger-api/cardano-ledger-api.cabal

```
@@ -40,7 +40,10 @@ library
         Cardano.Ledger.Api.UTxO
 
     hs-source-dirs:   src
-    other-modules:    Cardano.Ledger.Api.Scripts.ExUnits
+    other-modules:
+        Cardano.Ledger.Api.Scripts.ExUnits
+        Cardano.Ledger.Api.State.CommitteeMembersState
```

Could you move this module into a Query subfolder, since we'll have a lot more State, but not query related modules in the `Cardano/Ledger/Api/State` path
```suggestion
        Cardano.Ledger.Api.State.Query.CommitteeMembersState
```

---

Yes, sorry for forgetting, I know you told me yesterday. Done